common/sc_popen.cpp

```diff
@@ -104,7 +104,7 @@ sc_popen_argv(const char *filename, char *const argv[], pid_t *pidp, const char
 			(void)close(pdes[1]);
 		}
 
-		execve(filename, argv, environ);
+		execvpe(filename, argv, environ);
```

Comment:
already done -- see #3501

Reply:
Well, sc_popen_argv had not been changed in the same way.

Reply:
ah, i see.

Reply:
Ah, sorry, actually it had, but it is in 3.9... is that getting merged into develop soon ?